features/bewoning_zoeken_met_nummeraanduidingidentificatie.feature

```diff
@@ -0,0 +1,66 @@
+
+Scenario: Zoeken vanaf een datum
+    Als bewoningen wordt gezocht met nummeraanduiding identificatie 123456789 en datumvan 1993-01-01
+    Dan wordt de volgende bewoning teruggegeven
+    | naam openbare ruimte | huisnummer | identificatie code nummeraanduiding |
+    | Beethovenlaan        | 23         | 123456789                           |
+    En wordt bij de bewoning met identificatiecodenummeraanduiding 123456789 de volgende bewoners teruggegeven
+    | burgerservicenummer | datum aanvang adreshouding | datum tot  |
+    | 999993483           | 1992-03-05                 | 1994-04-30 |
+    | 999992338           | 1994-01-02                 | -          |
+    | 999994669           | 1994-05-08                 | -          |
+
+Scenario: Zoeken tot en met een datum
+    Als bewoningen wordt gezocht met nummeraanduiding identificatie 123456789 en datumtotenmet 1994-01-01
+    Dan wordt de volgende bewoning teruggegeven
+    | naam openbare ruimte | huisnummer | identificatie code nummeraanduiding |
+    | Beethovenlaan        | 23         | 123456789                           |
+    En wordt bij de bewoning met identificatiecodenummeraanduiding 123456789 de volgende bewoners teruggegeven
+    | burgerservicenummer | datum aanvang adreshouding | datum tot  |
+    | 999993483           | 1992-03-05                 | 1994-04-30 |
+
+Scenario: Zoeken in een periode
+    Als bewoningen wordt gezocht met nummeraanduiding identificatie 123456789, datumvan 1993-01-01 en datumtotenmet 1994-01-31
+    Dan wordt de volgende bewoning teruggegeven
+    | naam openbare ruimte | huisnummer | identificatie code nummeraanduiding |
+    | Beethovenlaan        | 23         | 123456789                           |
+    En wordt bij de bewoning met identificatiecodenummeraanduiding 123456789 de volgende bewoners teruggegeven
+    | burgerservicenummer | datum aanvang adreshouding | datum tot  |
+    | 999993483           | 1992-03-05                 | 1994-04-30 |
+    | 999992338           | 1994-01-02                 | -          |
+
+Scenario: Zoeken in een periode en peildatum (bewoner verblijft in periode en op peildatum op dezelfde verblijfplaats)
+    Als bewoningen wordt gezocht met nummeraanduiding identificatie 123456789, datumvan 1993-01-01 en datumtotenmet 1994-01-31 en peildatum 1994-04-01
+    Dan wordt de volgende bewoning teruggegeven
+    | naam openbare ruimte | huisnummer | identificatie code nummeraanduiding |
+    | Beethovenlaan        | 23         | 123456789                           |
+    En wordt bij de bewoning met identificatiecodenummeraanduiding 123456789 de volgende bewoners teruggegeven
+    | burgerservicenummer | datum aanvang adreshouding | datum tot  |
+    | 999993483           | 1992-03-05                 | 1994-04-30 |
+    | 999992338           | 1994-01-02                 | -          |
+
+Scenario: Zoeken in een periode en peildatum (bewoner verblijft in periode en op peildatum niet op dezelfde verblijfplaats)
```

Comment:
Ik zou hier zeggen: geen enkele bewoner verblijft zowel in opgegeven periode als op peildatum op de opgegeven verblijfplaats.

Reply:
klopt. aangepast